
package com.nortel.soa.oi.cct.connectionservice;

import javax.xml.ws.WebFault;


/**
 * This class was generated by Apache CXF 2.6.1
 * 2014-01-31T11:02:30.390+03:00
 * Generated source version: 2.6.1
 */

@WebFault(name = "RejectException", targetNamespace = "http://www.nortel.com/soa/oi/cct/faults")
public class RejectException extends Exception {
    
    private org.oasis_open.docs.wsrf._2004._06.wsrf_ws_basefaults_1_2_draft_01.ExceptionInfo rejectException;

    public RejectException() {
        super();
    }
    
    public RejectException(String message) {
        super(message);
    }
    
    public RejectException(String message, Throwable cause) {
        super(message, cause);
    }

    public RejectException(String message, org.oasis_open.docs.wsrf._2004._06.wsrf_ws_basefaults_1_2_draft_01.ExceptionInfo rejectException) {
        super(message);
        this.rejectException = rejectException;
    }

    public RejectException(String message, org.oasis_open.docs.wsrf._2004._06.wsrf_ws_basefaults_1_2_draft_01.ExceptionInfo rejectException, Throwable cause) {
        super(message, cause);
        this.rejectException = rejectException;
    }

    public org.oasis_open.docs.wsrf._2004._06.wsrf_ws_basefaults_1_2_draft_01.ExceptionInfo getFaultInfo() {
        return this.rejectException;
    }
}
